servers/Azure.Mcp.Server/README.md

```diff
@@ -1,13 +1,16 @@
-# <img height="36" width="36" src="https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/acom_social_icon_azure" alt="Microsoft Azure Logo" /> Azure MCP Server
+<!--
+See eng\scripts\Process-PackageReadMe.ps1 for instruction on how to anotate this README.md for package specific output
+-->
+# <!-- remove-section: start nuget;vsix --><img height="36" width="36" src="https://cdn-dynmedia-1.microsoft.com/is/content/microsoftcorp/acom_social_icon_azure" alt="Microsoft Azure Logo" /> <!-- remove-section: end -->Azure MCP Server <!-- insert-section: nuget;vsix;npm {{ToolTitle}} -->
```

Comment:
How does someone updating the main Readme now test that they haven't broken a generated package readme? 

Can the instructions be added to contributing.md with perhaps a comment in the main Readme pointing to the doc